Microsoft.Toolkit.Uwp.UI.Controls.DataGrid/DataGrid/DataGridComboBoxColumn.cs

```diff
@@ -637,5 +652,60 @@ private void EnsureItemsSourceBinding()
                 }
             }
         }
+
+        private void HookDataItemPropertyChanged(object dataItem)
+        {
+            var notifyingDataItem = dataItem as INotifyPropertyChanged;
```

Comment:
I had the lines 

```
            if (Binding.Mode == BindingMode.OneTime)
            {
                return;
            }
```

Is it wrong to skip the updates? I was assuming the binding was already done once.

Reply:
I actually had the same check in the first commit I did for this fix. When I then saw that Binding is hard coded to TwoWay and removed it. But, as you point out, it can be changed by the developer so it is probably good to include it as it would be the expected behavior.

Reply:
@RBrid Ready for review